loadtest/control/simplecontroller/actions.go

```diff
@@ -90,9 +90,20 @@ func (c *SimpleController) joinTeam() control.UserStatus {
 }
 
 func (c *SimpleController) createPost() control.UserStatus {
+	team, err := c.user.Store().RandomTeam()
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	channel, err := c.user.Store().RandomChannel(team.Id)
```

Comment:
A bit concerned about using random functions but I guess it will be interesting to see how this behaves.